src/Connection.cpp

```diff
@@ -288,7 +288,7 @@ void NetworkTable::Connection::ManageSocket(int timeout_millis) {
         // send/receive:
         init_socket.setsockopt(ZMQ_LINGER, 0);
         init_socket.setsockopt(ZMQ_RCVTIMEO, timeout_millis);
-        init_socket.connect("ipc:///tmp/sailbot/NetworkTable");
+        init_socket.connect("ipc://" + kWelcome_Directory_ + "NetworkTable");
```

Comment:
For Connection.py should we add a parameter to specify which directory when connecting?

Reply:
Im still thinking of a way to do it. I was hoping not to give anymore parameters to the client side code, but might have to